pkgs/applications/networking/instant-messengers/element/keytar/default.nix

```diff
@@ -1,10 +1,25 @@
-{ lib, stdenv, fetchFromGitHub, nodejs-14_x, python3, callPackage
-, fixup_yarn_lock, yarn, pkg-config, libsecret, xcbuild, Security, AppKit, fetchYarnDeps }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, nodejs-14_x
+, python3
+, callPackage
+, fixup_yarn_lock
+, yarn
+, pkg-config
+, libsecret
+, xcbuild
+, Security
+, AppKit
+, fetchYarnDeps
+, removeReferencesTo
+}:
```

Comment:
This doesn't seem related to the change. *If* we do this formatting, we should at least do it in a second commit.

Reply:
Thanks, I've undone that.